src/test/java/api/support/fixtures/RecordCreator.java

```diff
@@ -49,6 +49,10 @@ IndividualResource createIfAbsent(JsonObject record) {
     return createIfAbsent(identityMapKey.apply(record), record);
   }
 
+  void update(String servicePointCode, JsonObject updateRecord) {
+    identityMap.put(servicePointCode, client.create(updateRecord));
+  }
```

Comment:
This class looks to be very generic, it shouldn't know anything about service point codes